packages/ember-routing/lib/system/route.js

```diff
@@ -2150,7 +2117,7 @@ function buildRenderOptions(route, namePassed, isDefaultRender, _name, options)
     controller.set('model', options.model);
   }
 
-  let viewName = options && options.view || namePassed && name || route.viewName || name;
+  let viewName = namePassed && name || name;
   let ViewClass = owner._lookupFactory(`view:${viewName}`);
```

Comment:
I believe the whole thing here can be removed.  We shouldn't need any `view:*` lookups here...